Feature/be 4443 (#1)
appcircleio/appcircle-react-native-ui-test-component: README.md

# Appcircle React Native UI Test Component

> Run React Native UI Tests with Detox

Reviewer: ```suggestion
Run React Native UI tests with Detox.
```

## Required Input Variables

- `$AC_REPOSITORY_DIR`: Specifies the cloned repository directory.
- `$AC_OUTPUT_DIR`: Specify the output directory for test results.
- `$AC_RN_DETOX_CONFIGURATION`: Specify the Detox configuration name to used when building and running the tests.

> 

Reviewer: ```suggestion

## Optional Input Variables
- `$AC_RN_DETOX_ARGS`: ...
```

# Appcircle React Native UI Test Component

Run React Native UI tests with Detox.

## Required Input Variables

- `$AC_REPOSITORY_DIR`: Specifies the cloned repository directory.
- `$AC_OUTPUT_DIR`: Specify the output directory for test results.
- `$AC_RN_DETOX_CONFIGURATION`: Specify the Detox configuration name to used when building and running the tests.

## Optional Input Variables

> - `$AC_RN_DETOX_ARGS`: Specify the Detox extra commands to add the test command. Appcircle uses the command `detox build --configuration<configname>` command to be able to run Detox tests. The commands written here will be executed by appending detox build --configuration<configname> to the end of the command.

Reviewer: ```suggestion
- `$AC_RN_DETOX_ARGS`: Specify the Detox extra arguments to add the test command. The arguments will be executed by appending `detox build --configuration<configname>` to the end of the command. For more information, see the Detox test [CLI options](https://wix.github.io/Detox/docs/19.x/api/detox-cli/#test).
```
